.drone-v1.yml

```diff
@@ -0,0 +1,253 @@
+---
+kind: pipeline
+type: kubernetes
+name: build
+
+services:
+  - name: postgres
+    image: quay.io/ukhomeofficedigital/postgres-alpine
+    environment:
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: dev
+
+steps:
+  - name: build project
+    image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+    environment:
+      SPRING_PROFILES_ACTIVE: "development, local"
+      HOCS_DOCUMENT_SERVICE: "http://documents:8080"
+      DB_HOST: "postgres"
+      AWS_LOCAL_HOST: "localstack"
+    commands:
+      - ./gradlew build --no-daemon
```

Comment:
Let's split this into two steps: `./gradlew assemble --no-daemon` and `./gradlew check --no-daemon` (`gradlew build` is essentially an alias for these two things anyway).

Reply:
This has been completed